src/App.js

```diff
@@ -3,21 +3,13 @@ import './App.scss';
 
 import todos from './api/todos';
 import users from './api/users';
+import TodoList from './components/TodoList/TodoList';
 
 function App() {
   return (
-    <div className="App">
-      <h1>Static list of todos</h1>
-      <p>
-        <span>Todos: </span>
-        {todos.length}
-      </p>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
+    <main className="main">
+      <TodoList todos={todos} users={users} />
```

Comment:
do this instead of passing both arrays
> Create a preparedTodos array of todos with a user property added to each todo. Place it in the App.js

```suggestion
      <TodoList todos={preparedTodos} />
```